macros/GroupData.json

```diff
@@ -1333,14 +1346,41 @@
         "Presentation API": {
             "interfaces": [ "Presentation",
                             "PresentationAvailability",
+                            "PresentationConnection",
+                            "PresentationConnectionAvailableEvent",
+                            "PresentationConnectionCloseEvent",
+                            "PresentationConnectionList",
                             "PresentationDeviceInfoManager",
+                            "PresentationReceiver",
                             "PresentationRequest",
                             "PresentationSession",
                             "PresentationSessionConnectEvent" ],
-            "methods":    [],
+            "methods":    [ "PresentationConnection.close()",
+                            "PresentationConnection.send()",
+                            "PresentationConnection.terminate()",
+                            "PresentationRequest.start()",
+                            "PresentationRequest.reconnect()"
```

Comment:
[TravisCI noticed](https://travis-ci.org/mozilla/kumascript/builds/256945545#L1875-L1875) the missing comma.